### PR TITLE
Issue #2 search for activity by ID (key)

### PR DESCRIPTION
**Pull request by lsparlin**
## Addresses Issue
#2 

## Example
```
irb(main):003:0> Bored.now(key: 1878070).description
=> "Volunteer at your local food pantry"
```

### CODE REVIEW

File: test/test_bored.rb [hunk 1]
```diff
@@ -21,4 +21,22 @@ def test_that_it_gives_you_an_activity
...
+      :cooking, :relaxation, :music, :busywork
+    ], activity.type)
+  end
+
+  def test_that_it_gives_you_an_error_with_a_bad_key
+    activity = Bored.now(key: "bogus")
+
+    assert_kind_of String, activity.description
+    assert_equal "No activity found with the specified parameters", activity.description
+  end
```
Comment (searls) [inline test/test_bored.rb:41]: I think that providing a known key and getting a 404 would be an exceptional-enough condition that we might want to consider raising an error in this case. What do you think?

**Review by searls — APPROVED** [review]
Since this and #14 would conflict, I'd like to merge that one in first and then this one if that's ok.

Sorry for being so slow to review this! I didn't have notifications enabled 🤦

File: test/test_bored.rb (hunk 1 above)
Comment (lsparlin) [reply test/test_bored.rb:41]: Yeah I think you’re right. I wanted to give graceful feedback in the CLI, but I can defer that responsibility and raise an exception in the `Bored` class

**Comment by searls** [comment]
Landed in 0.1.0. Yay!! Thank you 🎉

**Pull request merged**